From 77eaddf4eb6b9e845131aa9e4e23b34459f471c0 Mon Sep 17 00:00:00 2001
From: Subrat Meher <smeher02.aoe3@gmail.com>
Date: Tue, 26 May 2020 11:07:21 +0200
Subject: [PATCH] Fix openzwave include

This is to allow finding openzwave files such as ValueIDIndexes.h
 which are located in /usr/include/openzwave and are included directly by name
 instead of with the openzwave prefix.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4d3daeec3ca0b63d8c1612afebeade1d8d756583..665ca565486738376869da8d65480c66edd65e66 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -713,7 +713,7 @@ IF(OpenZWave)
     IF(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
       include_directories(${OPENZWAVE_INCLUDE_DIRS})
     ELSE()
-      include_directories(${OPENZWAVE_INCLUDE_DIRS}/openzwave)
+      include_directories(${OPENZWAVE_INCLUDE_DIRS})
     ENDIF()
     message(STATUS "OpenZWave includes found at: ${OPENZWAVE_INCLUDE_DIRS}")
   else()
-- 
2.26.2

